wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+Ephemeral.swift

```diff
@@ -77,15 +77,12 @@ extension ConversationInputBarViewController {
         createEphemeralKeyboardViewController()
         ephemeralKeyboardViewController?.modalPresentationStyle = .popover
         ephemeralKeyboardViewController?.preferredContentSize = CGSize.IPadPopover.pickerSize
-        let pointToView: UIView = ephemeralIndicatorButton.isHidden ? hourglassButton : ephemeralIndicatorButton
+        let pointToView = ephemeralIndicatorButton.isHidden ? hourglassButton : ephemeralIndicatorButton
 
         if let popover = ephemeralKeyboardViewController?.popoverPresentationController,
-            let presentInView = self.parent?.view,
             let backgroundColor = ephemeralKeyboardViewController?.view.backgroundColor {
-                popover.config(from: self,
-                           pointToView: pointToView,
-                           sourceView: presentInView)
-
+            popover.sourceView = pointToView.superview!
+            popover.sourceRect = pointToView.frame.insetBy(dx: -4, dy: -4)
```

Comment:
Same here regarding force unwrap

Reply:
Also here I would consider the force-unwrapping safe, but I'm curious about other opinions.

I'm not sure if any edge case like animation/transition or whatever can result to the view's `superview` property being `nil` at the moment when this code is executed, but usually the button has a superview, since it's part of the view hierarchy.